dsc/cli.py

```diff
@@ -27,20 +28,33 @@
     "workflow run files",
     required=True,
 )
+@click.option(
+    "-r",
+    "--create-and-send-report",
+    is_flag=True,
+    help=(
+        "Pass to create and send summary report to recipients. "
+        "Reports generated for 'reconcile' and 'submit' steps must only be sent to "
+        "DSC admins."
```

Comment:
Even though I'd like to see this param shift to the specific CLI commands, I would also be less specific about who it is sent to, in `wiley` for example, the `submit` report would be sent to stakeholders as well. The first line seems sufficient

Reply:
Heartily agree.  I _almost_ commented elsewhere about docstring that was _almost_ too specific, making it somewhat brittle over time.  I do like detailed docstrings, but erring on the side of describing unusual mechanics over details that are subject to change.

Reply:
Updated.